flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/source/KafkaSourceITCase.java

```diff
@@ -26,6 +26,15 @@
 import org.apache.flink.configuration.Configuration;
 import org.apache.flink.connector.kafka.source.enumerator.initializer.OffsetsInitializer;
 import org.apache.flink.connector.kafka.source.reader.deserializer.KafkaRecordDeserializationSchema;
+import org.apache.flink.connector.kafka.source.testutils.KafkaMultipleTopicExternalContext;
```

Comment:
I don't quite understand the changes in this file. This looks to me if you have "only" converted the tests from JUnit4 to JUnit5. If so please move in some extra commit. I'd also like to see the reason of moving the tests inside a nested test class. Aren't all tests `KafkaSpecificTests`?

Reply:
The existing KafkaSourceITCase uses ```@BeforeAll``` and ```@AfterAll``` to startup a in-memory Kafka broker. If I only nest the testing framework cases, that in-memory Kafka broker will be running until all cases including testing framework cases finish. 

So I use the ```KafkaSpecificTests``` nested class to wrap all existing IT cases and destroy the in-memory Kafka before running testing framework cases. "KafkaSpecific" means these cases are testing Kafka source features like start timestamp, serializer etc.